src/js/mixins/evented.js

```diff
@@ -171,7 +191,7 @@ const normalizeListenArgs = (self, args) => {
  *         A listener function.
  */
 const listen = (target, method, type, listener) => {
-  validateTarget(target);
+  validateTarget(target, target, method);
```

Comment:
We never validated listen, which means a trigger on a disposed object or an invalid trigger would go through and give some strange errors further down in our `dom` helpers.